icons/folder-element.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" xml:space="preserve" viewBox="0 0 32 32"><path fill="#0288d1" d="m13.8 7.5-1.2-1a2 2 0 0 0-1.3-.5H4a2 2 0 0 0-2 2v16a2 2 0 0 0 2 2h24a2 2 0 0 0 2-2V10a2 2 0 0 0-2-2H15.1a2 2 0 0 1-1.3-.5Z"/><path fill="none" stroke="#b3e5fc" stroke-width="1.93" d="M30.64 28.54a10.8 3.41 45 0 1-10.01-5.23 10.8 3.41 45 0 1-5.24-10.01 10.8 3.41 45 0 1 10.02 5.23 10.8 3.41 45 0 1 5.23 10.01zm0-15.24a3.41 10.8 45 0 1-5.23 10.01 3.41 10.8 45 0 1-10.02 5.23 3.41 10.8 45 0 1 5.24-10 3.41 10.8 45 0 1 10.01-5.24z" paint-order="stroke fill markers"/></svg>
```

Comment:
The icon does not fit into the grid:

![Screenshot_2024-08-03-18-06-17-388_com.android.chrome.jpg](https://github.com/user-attachments/assets/1c186c39-7636-4976-ad66-d2b03ab09026)